pkgs/pkg-config/pkg-config.yaml

```diff
@@ -10,7 +10,7 @@ defaults:
 
 sources:
 - key: tar.bz2:yqfhovb74xdslg4bh5yn3lzctqtwbisq
-  url: http://rabbit.dereferenced.org/~nenolod/distfiles/pkgconf-0.9.6.tar.bz2
+  url: https://github.com/pkgconf/pkgconf/archive/pkgconf-0.9.6.tar.gz
```

Comment:
You need to also update the `key`. Use `hit fetch https://github.com/pkgconf/pkgconf/archive/pkgconf-0.9.6.tar.gz` to get it.